packages/sitecore-jss-nextjs/src/middleware/redirects-middleware.ts

```diff
@@ -143,6 +143,20 @@ export class RedirectsMiddleware extends MiddlewareBase {
         url.href = prepareNewURL.href;
       }
 
+      const normalizedHeaders = new Headers();
+
+      /**
+       * FIX for Netlify service
+       * Netlify doesn’t handle 301/302 redirects if Next.js sets the header x-middleware-next.
+       */
+      if (res?.headers) {
```

Comment:
Any additional unit tests to cover new scenarios?

Reply:
I've added